docs/windows.md

```diff
@@ -73,74 +90,15 @@ the Windows Nodes in the demo.
-
-```bash
-kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea.yml
-```
-
-Then, you can run the following commands. [nssm](https://nssm.cc/) will install
-Antrea as a Windows service. Please ensure `nssm` is on your machine, which is
-a handy tool to manage services on Windows. NOTE: `<KubernetesVersion>`,
-`<KubeconfigPath>` and `<KubeletKubeconfigPath>` should be set by you.
-`<KubeProxyKubeconfigPath>` is an optional parameter that is specific to
-kube-proxy mode. E.g.
-
-```powershell
-$InstallKubeProxy=$false # Only for kube-proxy mode
-$KubernetesVersion="v1.23.5"
-$KubeConfig="C:/Users/Administrator/.kube/config" # admin kubeconfig
-$KubeletKubeconfigPath="C:/etc/kubernetes/kubelet.conf"
-if ($InstallKubeProxy) { $KubeProxyKubeconfigPath="C:/Users/Administrator/kubeproxy.conf" }
-```
+* Install [containerd](https://learn.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=containerd#windows-server-1)
+  if not yet present.
 
-```powershell
-$TAG="v1.13.0"
-$KubernetesVersion="<KubernetesVersion>"
-$KubeConfig="<KubeconfigPath>"
-$KubeletKubeconfigPath="<KubeletKubeconfigPath>"
-if ($InstallKubeProxy) { $KubeProxyKubeconfigPath="<KubeProxyKubeconfigPath>" }
-$KubernetesHome="c:/k"
-$AntreaHome="c:/k/antrea"
-$KubeProxyLogPath="c:/var/log/kube-proxy"
+### Installation as a Pod
 
-curl.exe -LO "https://raw.githubusercontent.com/antrea-io/antrea/${TAG}/hack/windows/Helper.psm1"
-Import-Module ./Helper.psm1
-
-Install-AntreaAgent -KubernetesVersion "$KubernetesVersion" -KubernetesHome "$KubernetesHome" -KubeConfig "$KubeConfig" -AntreaVersion "$TAG" -AntreaHome "$AntreaHome"
-New-KubeProxyServiceInterface
-
-New-DirectoryIfNotExist "${AntreaHome}/logs"
-New-DirectoryIfNotExist "${KubeProxyLogPath}"
-# Install kube-proxy service
-if ($InstallKubeProxy) { nssm install kube-proxy "${KubernetesHome}/kube-proxy.exe" "--proxy-mode=userspace --kubeconfig=${KubeProxyKubeconfigPath} --log-dir=${KubeProxyLogPath} --logtostderr=false --alsologtostderr" }
-nssm install antrea-agent "${AntreaHome}/bin/antrea-agent.exe" "--config=${AntreaHome}/etc/antrea-agent.conf --logtostderr=false --log_dir=${AntreaHome}/logs --alsologtostderr --log_file_max_size=100 --log_file_max_num=4"
-
-nssm set antrea-agent DependOnService ovs-vswitchd
-if ($InstallKubeProxy) { nssm set antrea-agent DependOnService kube-proxy ovs-vswitchd }
-nssm set antrea-agent Start SERVICE_DELAYED_AUTO_START
-
-if ($InstallKubeProxy) { Start-Service kube-proxy }
-Start-Service antrea-agent
-```
-
-### Installation as a Pod (Docker/containerd based runtimes)
+This installation method requires Antrea 1.10 or higher, and containerd 1.6 or
+higher (containerd 1.7 or higher is recommended). It relies on support for
+[Windows HostProcess containers](https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/),
+which is generally available starting with K8s 1.26.
 
```

Comment:
Should we also introduce ovs containerization and its dependencies here?

Reply:
I am nor sure what that means?

Reply:
I meant running OVS as a Pod on a Windows node, but I see it has been updated in the `Antrea and OVS Management on Windows` section., please ignore it.